examples/vue3-vite-boilerplate/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "vite-vue-starter",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@telekom/scale-components": "^3.0.0-beta.110",
```

Comment:
We use the neutral version (`@telekom/scale-components-neutral`) in all examples so people is on the safe side legally when just "trying it out"

https://github.com/telekom/scale/blob/main/examples/vanilla/index.html#L13

Reply:
👍 updated